tests/acl/test_stress_acl.py

```diff
@@ -13,115 +16,143 @@
 
 LOOP_TIMES_LEVEL_MAP = {
     'debug': 10,
-    'basic': 500,
-    'confident': 5000
+    'basic': 50,
+    'confident': 200
 }
 
 # Template json file used to test scale rules
 STRESS_ACL_TABLE_TEMPLATE = "acl/templates/acltb_test_stress_acl_table.j2"
 STRESS_ACL_RULE_TEMPLATE = "acl/templates/acltb_test_stress_acl_rules.j2"
-STRESS_ACL_BASH_TEMPLATE = "acl/templates/acltb_test_stress_acl.sh"
+STRESS_ACL_READD_RULE_TEMPLATE = "acl/templates/acltb_test_stress_acl_readd_rules.j2"
 STRESS_ACL_TABLE_JSON_FILE = "/tmp/acltb_test_stress_acl_table.json"
 STRESS_ACL_RULE_JSON_FILE = "/tmp/acltb_test_stress_acl_rules.json"
-STRESS_ACL_BASH_FILE = "/tmp/acltb_test_stress_acl.sh"
-STRESS_ACL_BASH_LOG_FILE = "/tmp/acltb_test_stress_acl.log"
 
 LOG_EXPECT_ACL_TABLE_CREATE_RE = ".*Created ACL table.*"
 LOG_EXPECT_ACL_RULE_FAILED_RE = ".*Failed to create ACL rule.*"
 
+ACL_RULE_NUMS = 10
 
-@pytest.fixture(scope='module')
-def setup_stress_acl_table(rand_selected_dut, tbinfo, duthost):
-    mg_facts = rand_selected_dut.get_extended_minigraph_facts(tbinfo)
-    if tbinfo["topo"]["type"] == "mx":
-        table_port = duthost.acl_facts()["ansible_facts"]["ansible_acl_facts"]["DATAACL"]["ports"][0]
-    else:
-        table_port = list(mg_facts['minigraph_portchannels'].keys())[0]
-    if not table_port:
-        pytest.skip('No portchannels found')
 
+@pytest.fixture(scope='module')
+def prepare_test_file(rand_selected_dut):
     # Define a custom table type CUSTOM_TYPE by loading a json configuration
     rand_selected_dut.template(src=STRESS_ACL_TABLE_TEMPLATE, dest=STRESS_ACL_TABLE_JSON_FILE)
     rand_selected_dut.shell("sonic-cfggen -j {} -w".format(STRESS_ACL_TABLE_JSON_FILE))
-    # Create an ACL table and bind to Vlan1000 interface
-    cmd_create_table = "config acl add table STRESS_ACL L3 -s ingress -p {}".format(table_port)
-    cmd_remove_table = "config acl remove table STRESS_ACL"
-    loganalyzer = LogAnalyzer(ansible_host=rand_selected_dut, marker_prefix="stress_acl")
-    loganalyzer.load_common_config()
-
-    try:
-        logger.info("Creating ACL table STRESS_ACL with type L3")
-        loganalyzer.expect_regex = [LOG_EXPECT_ACL_TABLE_CREATE_RE]
-        # Ignore any other errors to reduce noise
-        loganalyzer.ignore_regex = [r".*"]
-        with loganalyzer:
-            rand_selected_dut.shell(cmd_create_table)
-    except LogAnalyzerError as err:
-        # Cleanup Config DB if table creation failed
-        logger.error("ACL table creation failed, attempting to clean-up...")
-        rand_selected_dut.shell(cmd_remove_table)
-        raise err
-
-    yield
-    logger.info("Removing ACL table STRESS_ACL")
-    # Remove ACL table
-    rand_selected_dut.shell(cmd_remove_table)
+    # Copy acl rules
+    rand_selected_dut.template(src=STRESS_ACL_RULE_TEMPLATE, dest=STRESS_ACL_RULE_JSON_FILE)
```

Comment:
The `STRESS_ACL_RULE_JSON_FILE` is not template. We don't need to use `dut.template`

Reply:
Sure, use dut.copy